### PR TITLE
doc: numbered eviction situations

### PR DESCRIPTION
**Pull request by CharlotteRose**
Signed-off-by: Scoots Hamilton <scoots@redhat.com>

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins render docs

**Comment by joscollin** [comment]
jenkins render docs

File: doc/cephfs/eviction.rst [hunk 1]
```diff
@@ -23,15 +23,15 @@ Automatic client eviction
 
 There are three situations in which a client may be evicted automatically:
```
Comment (joscollin) [inline doc/cephfs/eviction.rst]: I think dropping this colon would fix the issue.

**Comment by CharlotteRose** [comment]
jenkins redner docs

File: doc/cephfs/eviction.rst (hunk 1 above)
Comment (CharlotteRose) [reply doc/cephfs/eviction.rst]: I will edit that :) - thanks for the info ^_^

**Comment by joscollin** [comment]
jenkins render docs

**Comment by CharlotteRose** [comment]
jenkins render docs

**Pull request merged**